From: Simon McVittie <smcv@debian.org>
Date: Fri, 3 Mar 2017 10:53:52 +0000
Subject: Install gdb Python helpers as data, not as executable scripts

They do not start with the #!/usr/bin/python that would be necessary
to make them run with Python rather than a shell, and they would
not be useful to run anyway: they are libraries to be imported,
not scripts to be run.

Signed-off-by: Simon McVittie <smcv@debian.org>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=779522
Applied-upstream: 2.51.5, commit:e3f59e482
---
 glib/Makefile.am    | 2 +-
 gobject/Makefile.am | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/glib/Makefile.am b/glib/Makefile.am
index 82a7d3a2a..feee983c4 100644
--- a/glib/Makefile.am
+++ b/glib/Makefile.am
@@ -476,7 +476,7 @@ dist-hook: $(BUILT_EXTRA_DIST) $(top_builddir)/build/win32/vs9/glib.vcproj $(top
 
 # install gdb scripts
 gdbdir = $(datadir)/glib-2.0/gdb
-dist_gdb_SCRIPTS = glib_gdb.py
+dist_gdb_DATA = glib_gdb.py
 
 libglib-gdb.py: libglib-gdb.py.in
 	$(AM_V_GEN) $(SED) -e "s|\@datadir\@|$(datadir)|" $(srcdir)/libglib-gdb.py.in > $(builddir)/libglib-gdb.py
diff --git a/gobject/Makefile.am b/gobject/Makefile.am
index 948e86974..c1b74488d 100644
--- a/gobject/Makefile.am
+++ b/gobject/Makefile.am
@@ -261,7 +261,7 @@ distclean-local:
 
 # install gdb scripts
 gdbdir = $(datadir)/glib-2.0/gdb
-dist_gdb_SCRIPTS = gobject_gdb.py
+dist_gdb_DATA = gobject_gdb.py
 
 libgobject-gdb.py: libgobject-gdb.py.in
 	$(AM_V_GEN) $(SED) -e "s|\@datadir\@|$(datadir)|" $(srcdir)/libgobject-gdb.py.in > $(builddir)/libgobject-gdb.py
